packages/lodestar-cli/src/lodecli/cmds/beacon/cmds/run/options/logger.ts

```diff
@@ -1,98 +1,98 @@
 import {Options} from "yargs";
-import {LogLevels, LogLevel} from "@chainsafe/lodestar-utils/lib/logger";
+import {LogLevel, LogLevels} from "@chainsafe/lodestar-utils/lib/logger";
 
 export const logChain: Options = {
   alias: [
-    "log.chain",
-    "logger.chain",
+    "log.chain.level",
+    "logger.chain.level",
   ],
   hidden: true,
   type: "string",
   choices: LogLevels,
-  default: LogLevels[LogLevel.info],
+  default: LogLevel.info,
   group: "log",
 };
 
 export const logDb: Options = {
   alias: [
-    "log.db",
-    "logger.db",
+    "log.db.level",
+    "logger.db.level",
   ],
   hidden: true,
   type: "string",
   choices: LogLevels,
-  default: LogLevels[LogLevel.info],
+  default: LogLevel.info,
   group: "log",
 };
 
 export const logEth1: Options = {
   alias: [
-    "log.eth1",
-    "logger.eth1",
+    "log.eth1.level",
+    "logger.eth1.level",
   ],
   hidden: true,
   type: "string",
   choices: LogLevels,
-  default: LogLevels[LogLevel.info],
+  default: LogLevel.info,
   group: "log",
 };
 
 export const logNode: Options = {
   alias: [
-    "log.node",
-    "logger.node",
+    "log.node.level",
```

Comment:
You can prepend another alias "log.level"

Reply:
Oh yeah, thats true :D

Reply:
I'll do it in separate PR

Reply:
Actually, it won't work, because of default values for each module. "log.level" should change default values for all modules